mta.yaml

```diff
@@ -57,11 +61,14 @@ resources:
     service: xsuaa
     service-name: public-xsuaa-service
     service-plan: application
-- name: dm_dev
+- name: sdm-di
```

Comment:
You can keep the same instance name dm_dev
and replace the service name from sdm-dev to sdm.